client/js/main.js

```diff
@@ -53,7 +54,31 @@ $(document).ready(function() {
             ribbonModel.set('selectedIndex', 1)
     })
     
+
+    var halfWindowWidth = $(document).width() * 0.5;
+    var optionsFixedWidth = 400;
     var splitPanel  = new SplitPanel({el : "#main-view"})
+    splitPanel.getSection("main-table").initalise({ minWidth: 150, initialWidth: halfWindowWidth < (optionsFixedWidth + SplitPanelSection.sepWidth) ? (optionsFixedWidth + SplitPanelSection.sepWidth) : halfWindowWidth, level: 1});

```

Comment:
Yeah i agree, that is just an overlooked hang up from the way it was. When i started i tried to change as little as possible which included the index.html but i'll move the creation of the tag into the js.damoOn Feb 01, 2016, at 05:48 AM, Jonathon Love <notifications@github.com> wrote:In client/js/main.js:>      var splitPanel  = new SplitPanel({el : "#main-view"})
> +    splitPanel.getSection("main-table").initalise({ minWidth: 150, initialWidth: halfWindowWidth < (optionsFixedWidth + SplitPanelSection.sepWidth) ? (optionsFixedWidth + SplitPanelSection.sepWidth) : halfWindowWidth, level: 1});
is this a bit strange? creating the sections in index.html, and initializing them here? it means that changes always require changes in two places.what about both creating and initializing them here? (or you could use the 'data-*' attributes to control their initialization in index.html)what do you think?—Reply to this email directly or view it on GitHub.